Neos.Flow/Classes/Security/Authorization/Interceptor/RequireAuthentication.php

```diff
@@ -39,12 +39,13 @@ public function __construct(AuthenticationManagerInterface $authenticationManage
     }
 
     /**
-     * Invokes the the authentication, if needed.
+     * Invokes the authentication, if needed.
      *
      * @return boolean true if the security checks was passed
      */
     public function invoke()
     {
         $this->authenticationManager->authenticate();
+        return true;
```

Comment:
Hello dear reviewer, are the changes here correct?

Reply:
Yes, `authenticate()` throws, if it fails.